parent/pom.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Copyright (c) 2006 Google, Inc. All rights reserved.
+
+This program is licensed to you under the Apache License Version 2.0,
+and you may not use this file except in compliance with the Apache License Version 2.0.
+You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the Apache License Version 2.0 is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.google.inject</groupId>
+    <artifactId>guice-platform</artifactId>
+    <version>4.0.1-SNAPSHOT</version>
+    <relativePath>../platform</relativePath>
+  </parent>
+
+  <packaging>pom</packaging>
+  <artifactId>guice-parent</artifactId>
+
+  <name>Google Guice - Parent</name>
+  <description>Google Guice - Parent</description>
+
+  <properties>
+    <org.hamcrest.version>1.3</org.hamcrest.version>
+    <mockito-core.version>1.9.5</mockito-core.version>
+    <hsqldb-j5.version>2.0.0</hsqldb-j5.version>
+    <hsqldb-j5.version>2.0.0</hsqldb-j5.version>
+    <easymock.version>3.0</easymock.version>
+    <junit.version>4.11</junit.version>
+    <org.apache.felix.version>3.0.5</org.apache.felix.version>
+    <biz.aQute-bnd.version>0.0.384</biz.aQute-bnd.version>
+    <spring.version>3.0.5.RELEASE</spring.version>
+    <hibernate-entitymanager.version>4.1.7.Final</hibernate-entitymanager.version>
+  </properties>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.inject</groupId>
+        <artifactId>guice</artifactId>
+        <classifier>tests</classifier>
+        <version>${guice.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.google.inject</groupId>
+        <artifactId>guice-bom</artifactId>
```

Comment:
Seems odd to import the BOM into the build's parent pom, when it's conceptually an output of the build.